platform/winrt/os_winrt.cpp

```diff
@@ -482,28 +662,19 @@ OS::Time OSWinrt::get_time(bool utc) const {
 	return time;
 }
 
-OS::TimeZoneInfo OS_Windows::get_time_zone_info() const {
-	TIME_ZONE_INFORMATION info;
-	bool daylight = false;
-	if (GetTimeZoneInformation(&info) == TIME_ZONE_ID_DAYLIGHT)
-		daylight = true;
+OS::TimeZoneInfo OSWinrt::get_time_zone_info() const {
 
 	TimeZoneInfo ret;
-	if (daylight) {
-		ret.name = info.DaylightName;
-	} else {
-		ret.name = info.StandardName;
-	}
-
-	ret.bias = info.Bias;
+	ret.bias = -3;
+	ret.name = "BRT";
```

Comment:
Heh, did this just to make it build and them completely forgot to actually implement it...